### ISSUE TITLE
Wrong Template order

### ISSUE THREAD

**Issue by Zellwerker** [einpraegsam/powermail#459]
A project with a more complex template hierarchy showed that the TemplateUtility processes the RootPaths in raw order and not after the numbering.

```
[0] => EXT:powermail/Resources/Private/Layouts/
[1] => EXT:some_ext/Resources/Private/Extensions/Powermail/Layouts/
[2] => EXT:zw_news/Resources/Private/Extensions/Powermail/Layouts/
[100] => EXT:most_important/Resources/Private/Extensions/ZwKarriere/Powermail/Layouts/
[10] => EXT:less_important/Resources/Private/Powermail/Layouts/
```

The function `getTemplateFolders()` lacks of a `ksort($templatePaths, SORT_NUMERIC);` before `$templatePaths = array_values($templatePaths);`

Would you please add this?

```
    public static function getTemplateFolders($part = 'template')
    {
        $templatePaths = [];
        $extbaseConfig = self::getConfigurationManager()->getConfiguration(
            ConfigurationManagerInterface::CONFIGURATION_TYPE_FRAMEWORK,
            'powermail'
        );
        if (!empty($extbaseConfig['view'][$part . 'RootPaths'])) {
            $templatePaths = $extbaseConfig['view'][$part . 'RootPaths'];
            ksort($templatePaths, SORT_NUMERIC); // ----> THIS LINE
            $templatePaths = array_values($templatePaths);
        }
        if (empty($templatePaths)) {
            $templatePaths[] = 'EXT:powermail/Resources/Private/' . ucfirst($part) . 's/';
        }
        $templatePaths = array_unique($templatePaths);
        $absolutePaths = [];
        foreach ($templatePaths as $templatePath) {
            $absolutePaths[] = StringUtility::addTrailingSlash(GeneralUtility::getFileAbsFileName($templatePath));
        }
        return $absolutePaths;
    }
```

**Comment by einpraegsam** [comment]
Sounds logically to me. Feel free to make a pull request.

**Comment by Zellwerker** [comment]
https://github.com/einpraegsam/powermail/pull/460

**Comment by mschwemer** [comment]
Solved with #531
